api/can_predictions.py

```diff
@@ -0,0 +1,28 @@
+from typing import List, Optional
+import pydantic
+import datetime
+
+"""
+Documented https://www.dropbox.com/scl/fi/o4bec2kz8dkcxdtabtqda/CAN-V1-API-Draft.paper?dl=0&rlkey=f7elx3zmo6tt5s7mj5nvap9a0
+
+"""
+
+class _HospitalBeds(pydantic.BaseModel):
+    peakShortfall: int = pydantic.Field(default=0, description="Shortfall of beds needed at the peek hospitalizaitons")
+    peakDate: Optional[datetime.datetime] = pydantic.Field(default='', description="Date of peak hospitalizations")
+    shortageStartDate: Optional[datetime.datetime] = pydantic.Field(default='', description="Date when hospitals overload")
+
+
+class _Projections(pydantic.BaseModel):
+    aggregateDeaths: int = pydantic.Field(...)
+    hospitalBeds: _HospitalBeds = pydantic.Field(...)
+
+class CANPredictionAPIRow(pydantic.BaseModel):
```

Comment:
Alternate name idea: `CountyProjectionSummary` (not sure how i feel about summary).

I think having `CAN` in the name is a bit unnecessary (I've probably used it elsewhere tho)

Reply:
i like keeping API in there too but i can use projection although thtere is non predicrted data